### PR TITLE
list tasks which have passed their deadlines 

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: lib/hooks/update-action/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ module.exports = settings => {
         return Promise.resolve();
       }
 
-      const licenceHolder = await Profile.query().findById(project.licenceHolderId).eager('establishments');
+      const licenceHolder = await Profile.query().findById(project.licenceHolderId).withGraphFetched('establishments');
```
Comment (wheelsandcogs) [inline lib/hooks/update-action/index.js:35]: unrelated change -  this gets rid of a deprecation warning in the cli.

File: lib/middleware/profile.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ module.exports = settings => {
     }
     return Profile.query()
       .findOne({ userId: req.user.id })
-      .eager('[roles, establishments]')
+      .withGraphFetched('[roles, establishments]')
```
Comment (wheelsandcogs) [inline lib/middleware/profile.js:18]: unrelated change -  this gets rid of a deprecation warning in the cli.

File: lib/router/deadline-passed.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+    if (limit) {
+      query = Task.paginate({ query, limit, offset });
+    }
+
+    return query;
+  };
+
+  const checkPermissions = user => {
+    if (!user.profile.asruSupport) {
```
Comment (lennym) [inline lib/router/deadline-passed.js:47]:
```suggestion
    if (!user.profile.asruUser || !user.profile.asruSupport) {
```
